### ISSUE TITLE
Display only part of large text files

### ISSUE THREAD

**Issue by karissa** [datproject/datproject.org#511]
If the text file is really large, we can display only the first part of the files. Right now, files over 10MB just fail to render.

In csvs, we can display only 10 rows at a time, which would make file previews almost instant.

**Comment by karissa** [comment]
@yoshuawuyts didn't you say there's some cool demo with infinite scroll in choo v5 now? Can you point us to it? we could probably use it here!! :)

Maybe even could be a cool little element with a demo for dat compatibility!

**Comment by yoshuawuyts** [comment]
Yep, tim did some great work - https://github.com/timwis/hyperlist-component -
there might be some bugs still (see
https://github.com/yoshuawuyts/nanocomponent/issues/29) and the API might
need some work to be closer to
https://github.com/editdata/infinite-elements/issues/1 also. Oh, should
probs also mention https://github.com/yoshuawuyts/nanomorph/pull/61 which
should speed things up further. Hope this helps!
